### DISCUSSION TITLE
Search for multiple words in backend, in a grid column

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3903] [Questions & Answers]
Hi everybody,
I've a customer who'd like to search for product name in the backend, using multiple words but without having them to be in the exact order.
Something like an "explode" of the search query, and creating the query with `(name LIKE "%part1%" AND name LIKE "%part2%" AND name LIKE "%part% ")`

Do you know a module or something that provides this functionality?

**Comment by jparker1986** [comment]
Hey Fab,

I found this, not sure if it's of interest.

https://www.iksanika.com/products/advanced_admin_products_manager.html

**Reply by fballiano** [reply]
Thanks, I ended up overriding the preparecollection method of the grid, parsing the original search query and changing it ;-)

**Reply by addison74** [reply]
Filtering in grids is based on the exact order of the character string. For example, if you have a product "Adidas Men's Runner", you won't find it if you use adidas runner. This feature should be added in OpenMage to improve the productivity.

**Reply by fballiano** [reply]
to add it as a feature it would require so much more work that I can't do
